bot_management/bot_body.py

```diff
@@ -0,0 +1,126 @@
+from time import sleep, ctime
+import helpfull_functions
+from binance.client import Client
+from binance.enums import *
+from binance.exceptions import BinanceAPIException, BinanceOrderException
+from downloader.global_variables import *
+
+
+class Bot():
+
+    def __init__(self, bot_id):
+        self.running = True
+        self.bot_id = bot_id
+
+    def switcher(self, bot_id, binance_client, amount, what_to_do="waiting", test="0"):
+        """bot function that does all what bot does"""
+        client = helpfull_functions.setup_client("localhost", "8086", "test_bot_logs")
```

Comment:
Do we need initiate influx db client every time? I mean in every loop iteration of the bot? same with API_URL?